src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseDetails.java

```diff
@@ -180,7 +180,7 @@ public void setIncompleteCallbackResponse() {
 
     public boolean existsInData(CaseTypeTabField caseTypeTabField) {
         // Fields of type "Label" are a special case and should be allowed through regardless
-        return caseTypeTabField.getCaseField().getId().equals(LABEL_FIELD_TYPE) ||
+        return caseTypeTabField.getCaseField().getFieldType().getType().equals(LABEL_FIELD_TYPE) ||
```

Comment:
I'm 90% sure it's getFieldType().getType() here instead of 
getFieldType().getId()
What do you guys think?

Reply:
Confirming

Reply:
thanks @andrewfolga